.packit.yaml

```diff
@@ -0,0 +1,43 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-katello.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-katello.spec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: katello
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-katello
+
+upstream_tag_template: "{version}"
```

Comment:
Since I'm also trying to add packit to some project, I've got a question: is this line necessary? I mean, by https://packit.dev/docs/configuration/#upstream_tag_template it makes more sense if the tag contains something else along the version.

Reply:
It is not. It's just part of the template (as most projects/plugins use v{version}) and it felt more natural to set it "correctly" vs dropping